engines/glk/zcode/detection_tables.h

```diff
@@ -170,7 +170,7 @@ const PlainGameDescriptor ZCODE_GAME_LIST[] = {
 	{ "awitl",              "A Week in the Life" },
 	{ "ayac",               "Are You a Chef?" },
 	{ "azteca",             "Azteca" },
-	{ "b2demo",             "Pick up the Phone Booth and Die, Part 2" },
+	{ "b2demo",             "Pick up the Phone Booth and Die, Part 2 (Demo)" },
```

Comment:
For consistency with the other engines it'd be better to remove "demo" from the gameid and keep it in the
version field. I'd do something like this here
`{ "phoneboothp2",             "Pick up the Phone Booth and Die, Part 2" },`